x/ccv/provider/keeper/relay.go

```diff
@@ -422,6 +489,10 @@ func (k Keeper) EndBlockCCR(ctx sdk.Context) {
 			if currentTime.After(timeoutTimestamp) {
 				// vscTimeout expired
 				// stop the consumer chain and release unbondings
+				k.Logger(ctx).Info("removing timed out consumer chain - VSCPacket timed out",
```

Comment:
To make it clear this log is not executing in the middle of `StopConsumerChain`
```suggestion
				k.Logger(ctx).Info("about to remove timed out consumer chain - VSCPacket timed out",
```